TheCodeBlog/AppDelegate.swift

```diff
@@ -14,7 +14,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
```

Comment:
<!--
  1 failure:  Limit vertical wh...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: TheCodeBlog/AppDelegate.swift;
  Line: 15;
-->
- :no_entry_sign: Limit vertical whitespace to a single empty line. Currently 3. (`vertical_whitespace`)